.github/workflows/all.yml

```diff
@@ -307,13 +307,23 @@ jobs:
           Expand-Archive -Path ".\dist\k6-$env:VERSION-win64.zip" -DestinationPath .\packaging\
           move .\packaging\k6-$env:VERSION-win64\k6.exe .\packaging\
           rmdir .\packaging\k6-$env:VERSION-win64\
+      - name: Add signtool to PATH
+        run: echo "C:\Program Files (x86)\Windows Kits\10\bin\10.0.17763.0\x86" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
```

Comment:
This static path is concerning if it ever changes, but I haven't found a way to determine it dynamically. :-/

[This GitHub Action](https://github.com/dlemstra/code-sign-action/blob/master/index.ts#L15) also hardcodes it.

Reply:
I managed to get a more robust `${env:ProgramFiles(x86)}\Windows Kits\10\bin\x64` working (4353e64b), and also use the x64 executable @na--.